lib/std/system/comparisons.nim

```diff
@@ -106,6 +106,9 @@ template `>`*(x, y: untyped): untyped =
   ## "is greater" operator. This is the same as `y < x`.
   y < x
 
+type Orderable = concept
+  proc `<=`(x, y: Self): bool
+
 proc min*(x, y: int8): int8 {.noSideEffect, inline.} =
```

Comment:
Why do we still need these?

Reply:
Original system is the same having both these specified overloads and the generic one, also the implementation for float is different due to NaN. Shouldn't be needed otherwise